### PR TITLE
Some more Python spirit...

### PR DESCRIPTION
**Pull request by birkenfeld**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

Unfortunately we couldn't find an account corresponding
to your GitHub username at [bugs.python.org](http://bugs.python.org/) (b.p.o).
If you don't already have an account at b.p.o, please
[create one](http://bugs.python.org/user?@template=register) and make sure to
add your GitHub username. If you do already have an account at b.p.o then
please go there and under "Your Details" add your GitHub username.

And in case you haven't already, please make sure to sign the
[PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/)
(CLA); we can't legally look at your contribution until you have signed the
CLA.

Once you have done everything that's needed, please reply here and someone will
verify everything is in order.

**Comment by birkenfeld** [comment]
Strange, my GitHub username should be on record at b.p.o...

**Comment by brettcannon** [comment]
So should mine. Just pushed a change to log all the interactions with bugs.python.org to try and find out what's going on. Waiting for the bot to pick up on the change and restart.

Tracking in #29

File: ni/github.py [hunk 1]
```diff
@@ -22,22 +24,29 @@
...
 
-NO_USERNAME_BODY = """Unfortunately we couldn't find an account corresponding
-to your GitHub username at [bugs.python.org](http://bugs.python.org/) (b.p.o).
-If you don't already have an account at b.p.o, please
-[create one](http://bugs.python.org/user?@template=register) and make sure to
-add your GitHub username. If you do already have an account at b.p.o then
-please go there and under "Your Details" add your GitHub username.
+NO_CLA_BODY_EASTEREGG = NO_CLA_BODY + """
+
+We also demand... A SHRUBBERY!"""
```
Comment (brettcannon) [inline ni/github.py]: Link to the YouTube video of the sketch. :smile:

Comment (birkenfeld) [inline ni/github.py]: Fortunately it's available on the official Monty Python channel.

**Comment by brettcannon** [comment]
Fix the conflicts and then feel free to merge.

**Comment by birkenfeld** [comment]
I would, but it appears I can't :)

**Comment by brettcannon** [comment]
I thought I had made Python core committers on this repo but it turned out I didn't, probably to be overly cautious about testing coverage since the Heroku app deploys from master automatically.

**Pull request merged**